cli_tools/gce_vm_image_export/README.md

```diff
@@ -20,10 +20,12 @@ go get github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/gce_vm_image
   `pantheon`.
 + `-destination_uri=DESTINATION_URI` The Google Cloud Storage URI destination for the exported
   virtual disk file. For example: gs://my-bucket/my-exported-image.vmdk.
-+ `-source_image=SOURCE_IMAGE` An existing Compute Engine image URI from which to 
-  export.
 
-#### Optional flags  
+#### Optional flags
++ `-source_image=SOURCE_IMAGE` An existing Compute Engine image URI from which to
+  export. Exactly one of `-source_image` or `-source_disk_snapshot` must be provided.
++ `-source_disk_snapshot=SOURCE_DISK_SNAPSHOT` An existing Compute Engine disk snapshot URI from which to
+  export. Exactly one of `-source_image` or `-source_disk_snapshot` must be provided.
```

Comment:
These are not really optional flags, since at least one of them has to be specified.

Have a look at the README for https://github.com/GoogleCloudPlatform/compute-image-tools/tree/master/cli_tools/gce_vm_image_import
There, we have `--source_file` and `--source_image` that are similar to image/snapshot flags in the exporter. Use the same README structure for the exporter (under Required flags, add the "Exactly one of these must be specified:" sub-section).

Also, the usage section should be updated as well by adding image/snapshot flag group.

Reply:
Thank you for pointing out the example and the missing usage section.
README is updated.